src/Tools/CommonCoreClrRuntime/Class1.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace DoNotUse
```

Comment:
Comment as to why we have this do note use project in general.  Maybe put it in a README.MD in this directory.